app/utils.py

```diff
@@ -0,0 +1,14 @@
+from io import BytesIO
+
+from pypdf import PdfReader, PdfWriter
+
+
+def stitch_pdfs(first_pdf: BytesIO, second_pdf: BytesIO) -> BytesIO:
```

Comment:
I think we've dropped `@sentry_sdk.trace` off here in the rebase

Reply:
Ohh, re-added now 🙈